### ISSUE TITLE
[feature] Reporter Bundle Analyzer

### ISSUE THREAD

**Issue by Lagicrus** [parcel-bundler/parcel#6937]
# 🙋 feature request

A more in-depth look at the sizes of the various modules.

## 🤔 Expected Behavior

To be able to see the size of a module under various conditions such as this example
![image](https://user-images.githubusercontent.com/24227350/133452587-9944241b-4d55-409b-874d-7e6ae6c245fb.png)
When using webpack bundle analyzer

## 😯 Current Behavior

You only get raw byte size
![image](https://user-images.githubusercontent.com/24227350/133452690-f23fcc1b-c1ba-4e72-9212-12ec6fd2e983.png)

## 💁 Possible Solution

- Have a config file or options that a user can set to enable more in-depth settings
- Set default to make the data more human-readable i.e. 100000B becomes 100KB
- Show by default more in-depth details about the sizes
- Adding a configuration window like Webpack has, as shown below, allowing the user to change what is shown and in what sizes
![image](https://user-images.githubusercontent.com/24227350/133453265-ed735475-9e40-467f-a900-2af62206d1c3.png)

## 🔦 Context

Trying to best understand if Parcel is doing its job
How large actually are these files in a real-world scenario under various techniques

## 💻 Examples

Checkout how webpack does their bundle analyzer https://www.npmjs.com/package/webpack-bundle-analyzer

**Comment by mischnic** [comment]
Have you seen https://v2.parceljs.org/features/production/#analyzing-bundle-sizes?

**Comment by Lagicrus** [comment]
> Have you seen https://v2.parceljs.org/features/production/#analyzing-bundle-sizes?

Yes, I have, which is what lead to me writing this feature request.
- Detailed report is useful but useless for what I am looking for. It lists one size, not the breakdown I am looking for
- Same for the Bundle Analyzer which is where the second screenshot is from.
  - It lacks the customization I am looking for
  - There is no settings menu like the Webpack one, even though they both seem to use the same FoamTree
 - And Bundle-Buddy, I don't want to keep uploading my data to a website when I can't easily see what it is doing or the source code for it

I am looking for Parcel to enhance the features that it comes baked [with](https://github.com/parcel-bundler/parcel/tree/v2/packages/reporters/bundle-analyzer) and enhance them so it can compete and equal the offerings that Webpack offers, which is what the first and third screenshots come from
